### PR TITLE
[Feature-7419][Website][docs][doc-zh]Add unsubscribe method to website

### PR DESCRIPTION
**Pull request by uestczhuhao**
Add unsubscribe method to website both in docs and docs-zh.  Close apache/dolphinscheduler#7419

### CODE REVIEW

File: community/zh-cn/development/subscribe.md [hunk 1]
```diff
@@ -10,4 +10,16 @@
...
\ No newline at end of file
+3. 接收欢迎邮件。 完成以上步骤后，您会收到一封主题为WELCOME to dev@dolphinscheduler.apache.org的欢迎邮件，至此您已成功订阅Apache DolphinScheduler的邮件列表。
+
+# 取消订阅邮件列表
+
+如果您不再需要了解DolphinScheduler的动态，可以取消订阅邮件列表
+
+取消订阅邮件列表步骤如下:
+
+1. 用自己的邮箱向[dev-unsubscribe@dolphinscheduler.apache.org](mailto:dev-unsubscribe@dolphinscheduler.apache.org)发送一封邮件，主题和内容任意。
```
Comment (zhongjiajie) [inline community/zh-cn/development/subscribe.md:21]:
```suggestion
1. 用已经订阅的邮箱向[dev-unsubscribe@dolphinscheduler.apache.org](mailto:dev-unsubscribe@dolphinscheduler.apache.org)发送一封邮件，主题和内容任意。
```

File: community/zh-cn/development/subscribe.md [hunk 2]
```diff
@@ -10,4 +10,16 @@
...
+
+2. 接收确认邮件并回复。 完成步骤1后，您将收到一封来自dev-help@dolphinscheduler.apache.org的确认邮件（如未收到，请确认邮件是否被自动归入垃圾邮件、推广邮件、订阅邮件等文件夹）。然后直接回复该邮件，或点击邮件里的链接快捷回复即可，主题和内容任意。
+
+3. 接收告别邮件。 完成以上步骤后，您会收到一封主题为GOODBYE from dev@dolphinscheduler.apache.org 的告别邮件，至此您已成功取消订阅Apache DolphinScheduler的邮件列表。
```
Comment (zhongjiajie) [inline community/zh-cn/development/subscribe.md:25]:
```suggestion
3. 接收告别邮件。 完成以上步骤后，您会收到一封主题为GOODBYE from dev@dolphinscheduler.apache.org 的告别邮件，至此您已成功取消订阅Apache DolphinScheduler的邮件列表，以后将不会再接收来自 dev@dolphinscheduler.apache.org 的邮件通知
```

File: community/en-us/development/subscribe.md [hunk 3]
```diff
@@ -4,8 +4,20 @@ It is highly recommended to subscribe to the development mailing list to keep up
...
-1. Send an email to dev-subscribe@dolphinscheduler.apache.org with your own email address, subject and content.
+1. Send an email to dev-subscribe@dolphinscheduler.apache.org with your own email address, subject and content are arbitrary.
 
 2. Receive confirmation email and reply. After completing step 1, you will receive a confirmation email from dev-help@dolphinscheduler.apache.org (if not received, please confirm whether the email is automatically classified as spam, promotion email, subscription email, etc.) . Then reply directly to the email, or click on the link in the email to reply quickly, the subject and content are arbitrary.
 
 3. Receive a welcome email. After completing the above steps, you will receive a welcome email with the subject WELCOME to dev@dolphinscheduler.apache.org, and you have successfully subscribed to the Apache DolphinScheduler mailing list.
+
+# Unsubscribe Mailing Lists
+
+If you no longer need to know about DolphinScheduler's dynamics, you can unsubscribe from the mailing list
```
Comment (zhongjiajie) [inline community/en-us/development/subscribe.md:15]: I think this sentence should change, is a little odd. WDYT?

**Comment by zhongjiajie** [comment]
Hi @uestczhuhao I add some suggestion. PTAL

File: community/en-us/development/subscribe.md (hunk 3 above)
Comment (uestczhuhao) [reply community/en-us/development/subscribe.md:15]: How about change to:  If you do not need to know what's going on with DolphinScheduler, you can unsubscribe from the mailing list

**Review by dailidong — DISMISSED** [review]
+1
good job

File: community/en-us/development/subscribe.md (hunk 3 above)
Comment (dailidong) [reply community/en-us/development/subscribe.md]: > If you do not need to know what's going on with DolphinSc

I think it's better, good

**Comment by uestczhuhao** [comment]
> Hi @uestczhuhao I add some suggestion. PTAL

I have updated pr based on your suggestion, thank you

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
Thanks @uestczhuhao

**Pull request merged**